frontend/.env.example

```
@@ -14,4 +14,5 @@ GOOGLE_CLIENT_ID=
 GOOGLE_SPREADSHEET_ID=
 EXPO_PUBLIC_TRELLO_API_KEY=
 EXPO_PUBLIC_TRELLO_API_TOKEN=
-EXPO_PUBLIC_ADMIN_EMAIL=
\ No newline at end of file
+EXPO_PUBLIC_ADMIN_EMAIL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_ADMIN_EMAIL key should go before the EXPO_PUBLIC_FIREBASE_API_KEY key

(UnorderedKey)